packages/devtools_app/test/memory/memory_screen_test.dart

```diff
@@ -48,7 +48,7 @@ void main() {
   }
 
   void initControllerState() {
-    controller.offline.value = true;
+    controller.offline = true;
```

Comment:
seems like we shouldn't be relying on tests that test a code path that is now dead.